code/game/objects/items/shields.dm

```diff
@@ -193,7 +194,8 @@
 				if(QDELETED(flash) || flash.burnt_out)
 					return
 				playsound(src, 'sound/items/deconstruct.ogg', 50, TRUE)
-				qdel(embedded_flash)
+				if(embedded_flash)
+					qdel(embedded_flash)
```

Comment:
> `QDEL_NULL(null)` is safe, you shouldn't need to have to sanity check for it

It is not, it expands to
```
qdel(item)
item = null
```
Passing null results in a runtime of
```
[04:47:31] Runtime in garbage.dm,324: bad del
  proc name: qdel (/proc/qdel)
  src: null
  call stack:
  qdel(null, 0)
```

Reply:
@MrMelbert Oh you're right, I somehow managed to make it dispose before the variable was resolved in Initialized, and was still the initial type path. My apologies good sir.

This is why I personally dislike overloading variables as both paths and than the instance, but whatever. I've updated the intent of the changes to reflect that.

Reply:
I changed embedded_flash to a path to support map vareditted versions, but we can just change it back instead of doing weird !ispath() in Destroy memes lol

Reply:
This might ultimately be such a weird edge case that happened for me that it's probably not worth it.